src/main/java/org/gridsuite/modification/server/entities/equipment/modification/BranchModificationEntity.java

```diff
@@ -53,19 +52,89 @@ public class BranchModificationEntity extends BasicEquipmentModificationEntity {
+    })
+    private StringModificationEmbedded voltageLevelId2;
+
+    @Embedded
+    @AttributeOverrides(value = {
+        @AttributeOverride(name = "value", column = @Column(name = "busOrBusbarSectionId1")),
+        @AttributeOverride(name = "opType", column = @Column(name = "busOrBusbarSectionId1Op"))
+    })
+    private StringModificationEmbedded busOrBusbarSectionId1;
+
+    @Embedded
+    @AttributeOverrides(value = {
+        @AttributeOverride(name = "value", column = @Column(name = "busOrBusbarSectionId2")),
+        @AttributeOverride(name = "opType", column = @Column(name = "busOrBusbarSectionId2Op"))
+    })
+    private StringModificationEmbedded busOrBusbarSectionId2;
+
+    @Embedded
+    @AttributeOverrides(value = {
+        @AttributeOverride(name = "value", column = @Column(name = "connectionName1")),
+        @AttributeOverride(name = "opType", column = @Column(name = "connectionName1Op"))
+    })
+    private StringModificationEmbedded connectionName1;
+
+    @Embedded
+    @AttributeOverrides(value = {
+        @AttributeOverride(name = "value", column = @Column(name = "connectionName2")),
+        @AttributeOverride(name = "opType", column = @Column(name = "connectionName2Op"))
+    })
+    private StringModificationEmbedded connectionName2;
+
+    @Embedded
+    @AttributeOverrides(value = {
+        @AttributeOverride(name = "value", column = @Column(name = "connectionPosition1")),
+        @AttributeOverride(name = "opType", column = @Column(name = "connectionPosition1Op"))
+    })
+    private IntegerModificationEmbedded connectionPosition1;
+
+    @Embedded
+    @AttributeOverrides(value = {
+        @AttributeOverride(name = "value", column = @Column(name = "connectionPosition2")),
+        @AttributeOverride(name = "opType", column = @Column(name = "connectionPosition2Op"))
+    })
+    private IntegerModificationEmbedded connectionPosition2;
+
+    @Embedded
+    @AttributeOverrides(value = {
+        @AttributeOverride(name = "value", column = @Column(name = "connectionDirection1")),
+        @AttributeOverride(name = "opType", column = @Column(name = "connectionDirection1Op"))
+    })
+    private EnumModificationEmbedded<ConnectablePosition.Direction> connectionDirection1;
+
+    @Embedded
+    @AttributeOverrides(value = {
+        @AttributeOverride(name = "value", column = @Column(name = "connectionDirection2")),
+        @AttributeOverride(name = "opType", column = @Column(name = "connectionDirection2Op"))
+    })
+    private EnumModificationEmbedded<ConnectablePosition.Direction> connectionDirection2;
+
     @Embedded
     @AttributeOverrides(value = {
         @AttributeOverride(name = "value", column = @Column(name = "connected1")),
         @AttributeOverride(name = "opType", column = @Column(name = "connected1Op"))
     })
-    private BooleanModificationEmbedded connected1;
+    private BooleanModificationEmbedded terminal1Connected;
```

Comment:
change the AttributeOverride column name

Reply:
why ?

Reply:
well it is not synchronized with the attribute name but ok we do not change the schema